widgetssdk/src/main/java/com/glia/widgets/core/dialog/DialogController.java

```diff
@@ -109,13 +109,16 @@ public void showEnableCallNotificationChannelDialog() {
     }
 
     public void showEnableScreenSharingNotificationsAndStartSharingDialog() {
-            Logger.d(TAG, "Show Enable Notification Channel Dialog");
+        Logger.d(TAG, "Show Enable Notification Channel Dialog");
         dialogManager.offer(new DialogState(Dialog.MODE_ENABLE_SCREEN_SHARING_NOTIFICATIONS_AND_START_SHARING));
     }
 
     public void addCallback(Callback callback) {
         Logger.d(TAG, "addCallback");
         viewCallbacks.add(callback);
+        if (viewCallbacks.size() == 1) {
```

Comment:
Maybe it makes sense to check `if >= 1`?

Reply:
I think with `size() == 1` we are checking that this is the first callback added, if size > 1, it means that some view already handled that callback, and no need to trigger it again.

Reply:
Yeah. The idea is to show a dialog if the state was added before the callback registered.

Reply:
It might be good to add a comment about this I think. 😉